lib/controllr.rb

```diff
@@ -1,7 +1,11 @@
 require 'json'
+require 'geocoder'
+require 'redis'
 require './lib/age'
 
 class Controllr
+  OFFICE_ADDRESS = ENV['OFFICE_ADDRESS']
```

Comment:
You could get the office address from controllr also: `/api/system_settings.json`. See http://controllr.panter.biz/